src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos = [] }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo
+        key={todo.id}
+        title={todo.title}
+        completed={todo.completed}
+        user={todo.user}
```

Comment:
```suggestion
       key={todo.id}
       todo={todo}
```
Make it work like that